packages/react/src/components/checkbox/Checkbox.tsx

```diff
@@ -85,6 +101,11 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
         'Using ReactElement as a label is against good usability and accessibility practices. Please prefer plain strings.',
       );
     }
+    const tooltipProps = {
```

Comment:
Since it's used only once, I think we could drop the constant and spread the object instead 🤔